lib/src/typings/overrides.ts

```diff
@@ -2,24 +2,21 @@ import { useStyles as DayStyles } from '../views/Calendar/Day';
 import { useStyles as ClockStyles } from '../views/Clock/Clock';
 import { useStyles as MuiBasePickerStyles } from '../Picker/Picker';
 import { useStyles as CalendarStyles } from '../views/Calendar/Calendar';
-import { useStyles as MuiPickersYearStyles } from '../views/Calendar/Year';
+import { useStyles as MuiPickerYearStyles } from '../views/Calendar/Year';
```

Comment:
I think that we will need to rename the components to so they can be imported and grep as they were in a single namespace.

```suggestion
import { useStyles as MuiPickersYearStyles } from '../views/Calendar/PickerYear';
```